guides/micronaut-mqtt/cli/java/src/test/java/example/micronaut/MicronautguideCommandTest.java

```diff
@@ -4,52 +4,56 @@
 import io.micronaut.context.ApplicationContext;
 import io.micronaut.context.annotation.Replaces;
 import io.micronaut.context.env.Environment;
+import jakarta.inject.Singleton;
+import org.junit.jupiter.api.Test;
 
 import java.io.ByteArrayOutputStream;
+import java.io.OutputStream;
 import java.io.PrintStream;
 import java.math.BigDecimal;
-import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
-import jakarta.inject.Singleton;
-import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class MicronautguideCommandTest {
+class MicronautguideCommandTest {
 
     @Test
-    public void testWithCommandLineOption() throws Exception {
-        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    void testWithCommandLineOption() {
+        OutputStream baos = new ByteArrayOutputStream();
         System.setOut(new PrintStream(baos));
 
         try (ApplicationContext ctx = ApplicationContext.run(
                 Collections.singletonMap("spec.name", "MicronautguideCommandTest"),
```

Comment:
It needs to add `mqtt.enabled", false`. See https://github.com/micronaut-projects/micronaut-guides/pull/817/commits/1d422dc0aaa1a4df7e2cf07df5fa7ea7a249bd40